content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -152,7 +152,7 @@ You will still have to enable each option using the `CPUManagerPolicyOptions` ku
 
 The following policy options exist for the static `CPUManager` policy:
 * `full-pcpus-only` (beta, visible by default) (1.22 or higher)
-* `distribute-cpus-across-numa` (alpha, hidden by default) (1.23 or higher)
+* `distribute-cpus-across-numa` (beta, visible by default) (1.23 or higher)
```

Comment:
```suggestion
* `distribute-cpus-across-numa` (beta, visible by default) (1.33 or higher)
```

Reply:
IMO, we should mention 1.33 instead of v1.23, as `distribute-cpus-across-numa` will be in beta.

Reply:
Thanks, will update to 1.33.